### PR TITLE
Choose restaurant and list food orders

### PR DESCRIPTION
**Pull request by TNagaraja**
Issue #3, Issue #15

### CODE REVIEW

File: wfl/Database/MongoDBManager.cs [hunk 1]
```diff
@@ -32,5 +32,32 @@ public MongoDBManager()
...
+        public async Task<IEnumerable<FoodOrderDto>> GetAllFoodOrdersAsync()
+        {
+            IMongoCollection<FoodOrderDto> collection = db.GetCollection<FoodOrderDto>("food-orders");
+            IAsyncCursor<FoodOrderDto> task = await collection.FindAsync(order => true, null);
+            return task.ToEnumerable<FoodOrderDto>();
+        }
+
+        public async Task InsertFoodOrderAsync(FoodOrderDto order)
+        {
+            order.Id = MongoDB.Bson.ObjectId.GenerateNewId().ToString();
```
Comment (JChapman202) [inline wfl/Database/MongoDBManager.cs]: why are we generating a new id?  Can't we just let the database generate it like we did for restaurant?

File: wfl/Models/FoodOrder.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Symphono.Wfl.Models
+{
+    public class FoodOrder
+    {
+        public string Id { get; set; }
```
Comment (JChapman202) [inline wfl/Models/FoodOrder.cs]: Did we use strings or ObjectId for the Restaurant object?  I'm fine with either, but we need to make sure they're consistent.

File: wfl/Database/MongoDBManager.cs [hunk 3]
```diff
@@ -32,5 +32,32 @@ public MongoDBManager()
...
+            IMongoCollection<FoodOrderDto> collection = db.GetCollection<FoodOrderDto>("food-orders");
+            await collection.InsertOneAsync(order);
+        }
+
+        public async Task<bool> CheckRestaurantIdAsync(string id)
```
Comment (JChapman202) [inline wfl/Database/MongoDBManager.cs:58]: Not part of this PR, but I think we'll spend some time talking about design patterns we want to use moving forward.  There is going to start being a lot of methods in this class.

**Review by JChapman202 — COMMENTED** [review]
This is a good start, and I think we're going to start progressing really quickly.  Excited to see what we can get done in the next week.

File: wfl/Database/MongoDBManager.cs (hunk 1 above)
Comment (TNagaraja) [reply wfl/Database/MongoDBManager.cs]: Restaurant Id is being generated the same way currently. The reason why I removed the mongoDB attributes from the Restaurant and FoodOrder classes was to separate the database specific code from our models. I'm not sure if I can have the database generate Id's without placing those attributes back again.

File: wfl/Models/FoodOrder.cs (hunk 2 above)
Comment (TNagaraja) [reply wfl/Models/FoodOrder.cs]: The Restaurant object is currently using strings as well.

File: wfl/Database/MongoDBManager.cs (hunk 3 above)
Comment (TNagaraja) [reply wfl/Database/MongoDBManager.cs:58]: Sounds good.

File: wfl/Database/MongoDBManager.cs (hunk 1 above)
Comment (TNagaraja) [reply wfl/Database/MongoDBManager.cs]: Changing back to using attributes

**Comment by JChapman202** [comment]
Looks like merging the other PR caused this to have a conflict which will need to be resolved now before this can be merged.

**Comment by TNagaraja** [comment]
Merge conflicts should be resolved now.
